File: launchdarkly/python/src/GetBigSegmentImportExample.py
import json
from datetime import date, datetime
from pprint import pprint

from launchdarkly_client import ApiClient, ApiException, Configuration, api, models

configuration = Configuration(
    api_key={"ApiKey": "YOUR_API_KEY"},
)

with ApiClient(configuration) as api_client:
    try:
        response = api.SegmentsApi(api_client).get_big_segment_import(
            project_key=None,
            environment_key=None,
            segment_key=None,
            import_id=None,
        )

        pprint(response)
    except ApiException as e:
        print("Exception when calling Segments#get_big_segment_import: %s\n" % e)

File: launchdarkly/php/src/GetBigSegmentImportExample.php
<?php

namespace OSEG\LaunchDarklyExamples;

require_once __DIR__ . '/../vendor/autoload.php';

use SplFileObject;
use LaunchDarkly;

$config = LaunchDarkly\Client\Configuration::getDefaultConfiguration();
$config->setApiKey("Authorization", "YOUR_API_KEY");

try {
    $response = (new LaunchDarkly\Client\Api\SegmentsApi(config: $config))->getBigSegmentImport(
        project_key: null,
        environment_key: null,
        segment_key: null,
        import_id: null,
    );

    print_r($response);
} catch (LaunchDarkly\Client\ApiException $e) {
    echo "Exception when calling SegmentsApi#getBigSegmentImport: {$e->getMessage()}";
}
